src/device/selection.ts

```diff
@@ -375,6 +375,21 @@ export const split = (el: HTMLElement): SplitResult | null => {
     right: rightDiv.innerHTML,
   }
 }
+/** Removes the current selection. */
+export const removeCurrentSelection = () => {
+  const selection = window.getSelection()
+  if (selection && selection.rangeCount > 0) document.execCommand('delete')
+}
 
 /** Returns the selection text, or null if there is no selection. */
 export const text = () => window.getSelection()?.toString() ?? null
+
+/** Returns the selection html, or null if there is no selection. */
+export const html = () => {
+  const selection = document.getSelection()
+  const range = selection?.getRangeAt(0)
+  const div = document.createElement('div')
+  div.appendChild(range!.cloneContents())
```

Comment:
Return `null` if range is undefined. `!` is usually used when there are other runtime conditions that guarantee the value is truthy, and should be used sparingly.